mantle/kola/cluster/cluster.go

```diff
@@ -157,7 +157,9 @@ func (t *TestCluster) MustSSH(m platform.Machine, cmd string) []byte {
 	if err != nil {
 		if t.SSHOnTestFailure() {
 			plog.Errorf("dropping to shell: %q failed: output %s, status %v", cmd, out, err)
-			platform.Manhole(m)
+			if err := platform.Manhole(m); err != nil {
```

Comment:
Ignoring errors here might have been semi-intentional since the main error we care about is from the test, not from our SSH connection to debug it.  But eh.

Reply:
Thank you for review. Are you suggesting to exclude the error check in golang-ci lint since ignoring those errors are intentional?